Resilio Connect API/JSMonitor/app.js

```diff
@@ -36,3 +36,23 @@ setTimeout(function() {console.log(enumerateAgents());}, 5000);
 
 console.log(findArrayDiff([1, 5, 17, 149, 150, 151, 152, 153, 154, 155, 156, 157, 158, 159, 160], [1, 5, 17, 150, 151, 152, 153, 154, 155, 156, 157, 158, 159, 160, 165]));
 
+var prevList;
+var updatedList = [];
+
+function periodicAgentUpdate() {
+    prevList = updatedList; 
+    updatedList = enumerateAgents();
+
+    if (findArrayDiff(updatedList, prevList).length != 0) {
+        console.log("ALERT: " + findArrayDiff(updatedList, prevList));
```

Comment:
It is better not to call the same function twice.  It is a waste fo cpu time.
Call it once and use the result when you need it.